lua/nvim-web-devicons.lua

```diff
@@ -240,10 +240,22 @@ local icons_by_filename = {
     cterm_color = "185",
     name = "Favicon",
   },
+  ["GNUmakefile"] = {
+    icon = "",
+    color = "#ffffff",
```

Comment:
Unfortunately we can't use white, as that won't work for white background terminals.

We could just keep the existing colour, as that seems to work for light and dark terminals.